dataflow/src/main/java/org/checkerframework/dataflow/analysis/AnalysisResult.java

```diff
@@ -337,6 +337,16 @@ public S getStoreBefore(Block block) {
     }
   }
 
+  /**
+   * Returns the TransferInput for the given block.
+   *
+   * @param block a CFG Block
+   * @return the TransferInput for the given block.
+   */
+  public @Nullable TransferInput<V, S> getInput(Block block) {
+    return inputs.get(block);
+  }
```

Comment:
Because previously, the CalledMethods analysis was still running (since rlc subclassed cmc) and could be queried for CalledMethods stores at the time of the consistency analysis. Now, it's finished at the time of the consistency analysis and the analysisresult must be queried. It didn't allow querying for blocks, so I had to add this to have an equivalent querying method for analysis and analysisresult.